devel/libnick/Portfile

```diff
@@ -9,16 +9,16 @@ PortGroup           openssl 1.0
 
 boost.version       1.81
 
-github.setup        NickvisionApps libnick 2024.11.1
+github.setup        NickvisionApps libnick 2024.12.0
 revision            0
 categories          devel
 license             GPL-3
-maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+maintainers         {@nlogozzo} openmaintainer
```

Comment:
why this change; was that discussed somewhere?

Reply:
_This is the upstream._

And yes, see: https://github.com/NickvisionApps/Parabolic/issues/973#issuecomment-2501584866

Reply:
okay, but don't expect me to follow upstream discussions to go and figure this out ;)